CHANGELOG.md

```diff
@@ -10,7 +10,14 @@ We've updated the Accordion component to use the new [`hidden="until-found"` att
 
 This allows the browser's native 'find in page' functionality to search within and automatically open sections of the accordion. Currently, this functionality is only supported by recent versions of Google Chrome, Microsoft Edge and Samsung Internet.
 
-This was added in [pull request #3053: Enhance the Accordion component with `hidden='until-found'`](https://github.com/alphagov/govuk-frontend/pull/3053). 
+This was added in pull requests:
+
+- [#3053: Enhance the Accordion component with `hidden='until-found'`](https://github.com/alphagov/govuk-frontend/pull/3053)
+- [#3095: Hide Accordion content (again) during `.js-enabled` page load](https://github.com/alphagov/govuk-frontend/pull/3053)
+
+Note: To improve the reading experience on slow loading pages we initially removed `display: none` from each Accordion section. To avoid a flicker of content, we've chosen to restore the current behaviour in [pull request #3095](https://github.com/alphagov/govuk-frontend/pull/3053).
```

Comment:
I don't think this line is necessary. You already reference this PR on line 16 and I think providing the link is enough.